.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,29 @@
+### All submissions:
+
+* [ ] Does your code follow
+  the [WordPress' coding standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/)?
+* [ ] Have you checked to ensure there aren't other open [Pull Requests](../../pulls) for the same update/change?
+
+<!-- Mark completed items with an [x] -->
```

Comment:
I feel like this `All submissions` section and checklist is boilerplate that we will skip over - is it providing value? Do we get enough contributor PR submissions that this is worthwhile?

Keen to hear thoughts from the rest of the squad.

Reply:
Maybe it's worth linking to the `CONTRIBUTING.md` file instead of the WP standards? e.g.

```
* Does your code follow the [contributing guidelines](https://github.com/woocommerce/facebook-for-woocommerce/blob/master/.github/CONTRIBUTING.md)?
```

Or we just rely on GitHubs own warning:

<img width="904" alt="Screen Shot 2021-04-23 at 5 11 39 pm" src="https://user-images.githubusercontent.com/1892137/115836744-02bd4300-a457-11eb-8a7b-50f82cfc7cce.png">


> Have you checked to ensure there aren't other open Pull Requests?

I think this is implied and _ideally_ we won't have so many open PRs that this becomes something contributors need to watch out for.